tests/entropy_initial_lookup.prm

```diff
@@ -0,0 +1,145 @@
+#
+
+set Additional shared libraries            = $ASPECT_SOURCE_DIR/benchmarks/entropy_adiabat/plugins/libentropy_plugins.so
```

Comment:
This line will not work in the test. If you indeed need those plugins for this test, you need to instead add a .cc file with the same name as the test, and include all .cc files that you need inside that file. (see entropy_adiabat.cc in the same folder).

Reply:
Fixed